### PR TITLE
feat : added modal and navigation

### PR DESCRIPTION
**Pull request by rohit8686**
Modal - [Link](https://spiralui.netlify.app/documentation.html#modal)
Navigation - [Link](https://spiralui.netlify.app/documentation.html#navigation)

Contains :

- modal 
- responsive nav bar

### CODE REVIEW

File: Components/Modal/modal.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      <div class="flex">
+        <button class="btn open-modal btn-error">Open Modal</button>
+      </div>
+      <div class="modal-bg modal-hidden flex">
+        <div class="modal">
+          <div class="flex space-between">
+            <h2>Modal Heading</h2>
+            <span class="material-icons modal-close">close</span>
+          </div>
+          <br />
```
Comment (romabulani) [inline Components/Modal/modal.html:26]: Use of break tag is not semantically good, if possible avoid it

**Review by romabulani — APPROVED** [review]
Rest all looks good to merge

**Comment by ravikumar1002** [comment]
> _No description provided._

Provide description

**Review by ravikumar1002 — APPROVED** [review]
good to merge

**Pull request merged**